Gulpfile.coffee

```diff
@@ -25,7 +25,7 @@ gulp.task "test", ["stylecheck"], ->
   gulp.src(
     "tests/spec/*.coffee"
   ).pipe(mocha(
-    "timeout": 10000
+    "timeout": 30000
```

Comment:
Well, with the 10000 timeout, Travis [ran into a timeout](https://travis-ci.org/hiroaki-yamamoto/node-validator-nu/builds/79910968). Might be that 15000 would be enough, but on the other hand, what's the problem with this timeout? You *have* to expect JVM startup to take considerable time on many architectures, and the way I understand it, v.nu gets some schema files from the web, which will take some time as well, depending on the internet connection. I'd rather play this safe, and perhaps [explicitely reduce timeouts](http://mochajs.org/#timeouts) for the `remote` suite which should be faster if there is any real reason for doing so.